pkg/storaged/filesystem/utils.jsx

```diff
@@ -110,8 +108,8 @@ export function get_cryptobacking_noauto(client, block) {
     return crypto_options.indexOf("noauto") >= 0;
 }
 
-export const MountPoint = ({ fstab_config, forced_options, backing_block, content_block }) => {
-    const is_filesystem_mounted = content_block && is_mounted(client, content_block);
+export const MountPoint = ({ fstab_config, forced_options, backing_block, content_block, subvol }) => {
```

Comment:
The MountPoint component needs to pass `subvol` to `mounting_dialog` below, with the "edit" button.

Reply:
It does now.